src/App.js

```diff
@@ -33,7 +33,49 @@ const App = () => {
 
       // ACTION ITEM: your Pig Latin logic goes here!
 
-    
+      // Create a function
+      // Create a conditional that takes the first index of each word in the string
+      // See if its strictly equal to a vowel.
+      // Returns if first index is equal to a vowel return string + way
+
+      const firstVowel = () => {
+        if (eachWord[0] === 'a') {
+          return eachWord + "way"
+        } else if (eachWord[0] === 'e') {
+          return eachWord + "way"
+        } else if (eachWord[0] === 'i') {
+          return eachWord + "way"
+        } else if (eachWord[0] === 'o') {
+          return eachWord + "way"
+        } else if (eachWord[0] === 'u') {
+          return eachWord + "way"
+        }
+      } 
+      return firstVowel()
```

Comment:
This return ends the function. First vowel doesn't need to be found using a function if it's already available via vowelsArray[0]

Reply:
We haven't figured out how to access vowels array and make it work. We hard coded it and it worked. We want to make it work with vowels array but still trying.